pkg/types/spec/validations.go

```diff
@@ -165,6 +166,18 @@ func podValidation() *cr.StructFieldValidation {
 						},
 					},
 				},
+				{
+					StructField: "Port",
+					Int32PtrValidation: &cr.Int32PtrValidation{
+						Required:          false,
+						Default:           nil,
```

Comment:
nit: add a comment that this isn't used for task, which is why it's a pointer and doesn't have a default